src/init/bitcoin-gui.cpp

```diff
@@ -35,6 +35,7 @@ class BitcoinGuiInit : public interfaces::Init
     std::unique_ptr<interfaces::Echo> makeEcho() override { return interfaces::MakeEcho(); }
     interfaces::Ipc* ipc() override { return m_ipc.get(); }
     node::NodeContext m_node;
+    bool canListenIpc() override { return true; }
```

Comment:
Just a question: Intuitively I was under the impression that only the node should be listening. In which scenario should the gui be able to listen as well? Do we need a bi-directional connection here for some callbacks?